crawler/votacoes/aderencia/analyzer_aderencia.R

```diff
@@ -6,21 +6,28 @@
 #' @param parlamentares_path Caminho para o arquivo de dados de parlamentares
 #' @param proposicoes_url URL para a tabela de proposições com informações dos temas no VA
 #' @param casa_aderencia Casa para o cálculo da aderência (pode ser "camara" ou "senado)
+#' @param filtro Flag para expressar se deseja somente as proposições selecionadas
 #' @return Dataframe com informações de aderência
 processa_aderencia_parlamentares <-
   function(votos_path = here::here("crawler/raw_data/votos.csv"),
            orientacoes_path = here::here("crawler/raw_data/orientacoes.csv"),
            parlamentares_path = here::here("crawler/raw_data/parlamentares.csv"),
            proposicoes_url = NULL,
-           casa_aderencia = "camara") {
+           casa_aderencia = "camara",
+           filtro = 1) {
     library(tidyverse)
     library(here)
+  
     source(here("crawler/votacoes/utils_votacoes.R"))
+    source(here("crawler/votacoes/fetcher_votacoes_camara.R"))
     source(here("crawler/votacoes/aderencia/processa_dados_aderencia.R"))
     source(here("crawler/parlamentares/partidos/utils_partidos.R"))
     source(here("crawler/proposicoes/fetcher_proposicoes_senado.R"))
+    source(here("crawler/proposicoes/analyzer_proposicoes.R"))
     source(here("crawler/proposicoes/utils_proposicoes.R"))
     source(here("crawler/proposicoes/process_proposicao_tema.R"))
+    source(here("crawler/proposicoes/fetcher_proposicao_info.R"))
+    source(here("crawler/proposicoes/fetch_proposicoes_voz_ativa.R"))
     
     ## Preparando dados de votos, orientações e senadores
     votos <-
```

Comment:
Tanto votos como orientações devem ser filtrados para quando o parametro do filtro for 1. O filtro envolve considerar apenas os votos e as orientações das proposições selecionadas